colpivot.sql

```diff
@@ -103,8 +109,10 @@ create or replace function colpivot(
             foreach col in array key_cols loop
                 if i > 0 then
                     on_e := on_e || ' and ';
+                    on_e := on_e || '(' || ali || '.' || quote_ident(col) || ' = _key.' || quote_ident(col) || ' OR (' || ali || '.' || quote_ident(col) || ' IS NULL AND _key.' || quote_ident(col) || ' IS NULL)) ';
+                else
+                    on_e := on_e || ali || '.' || quote_ident(col) || ' = _key.' || quote_ident(col) || ' ';
```

Comment:
Why does the first column not get the null comparison check? It's probably obvious but it was so long that I wrote this code.